lib/hook.js

```diff
@@ -62,22 +62,18 @@ Hook.prototype.error = function(err) {
  */
 Hook.prototype.serialize = function serialize() {
   return {
-    $$isPending: this.isPending(),
+    $$isPending: !!this.isPending(),
     $$titlePath: this.titlePath(),
     ctx:
       this.ctx && this.ctx.currentTest
         ? {
-            currentTest: {
-              title: this.ctx.currentTest.title,
-              [MOCHA_ID_PROP_NAME]: this.ctx.currentTest.id
-            }
+            currentTest: this.ctx.currentTest.serialize()
           }
         : {},
-    parent: {
-      [MOCHA_ID_PROP_NAME]: this.parent.id
-    },
+    parent: this.parent.serialize(),
```

Comment:
Adding `$$fullTitle: this.parent.fullTitle()` should be sufficient.

Reply:
Done.